src/v2/Apps/Order/Routes/Review/index.tsx

```diff
@@ -383,9 +388,19 @@ export class ReviewRoute extends Component<ReviewProps> {
               <Flex flexDirection="column">
                 <Flex flexDirection="column">
                   <ArtworkSummaryItem order={order} />
-                  <TransactionDetailsSummaryItem order={order} />
+                  <TransactionDetailsSummaryItem
+                    order={order}
+                    placeholderOverride={
+                      isMissingMetadata ? "To be confirmed*" : null
+                    }
+                  />
                 </Flex>
                 <BuyerGuarantee />
+                {isMissingMetadata && (
+                  <Text variant="small" color="black60">
+                    *Shipping and taxes to be confirmed by gallery
```

Comment:
It's possible only one of shipping cost and tax is missing, so we can probably separate them out and check the presence of `shippingTotalCents` and `taxTotalCents` individually.

Reply:
I have a [discussion](https://www.figma.com/file/JE8dGuL81Kqd06R4egxdZo?node-id=118%3A684&fuid=825389106351956176#79189851) with @ArvindLall to make the messaging generic, and we can iterate on this. 👍